lib/auth/api.go

```diff
@@ -1135,6 +1135,8 @@ type Cache interface {
 	// implicit member list and the underlying implementation does not have
 	// enough information to compute the dynamic member list.
 	ListAccessListMembers(ctx context.Context, accessListName string, pageSize int, pageToken string) (members []*accesslist.AccessListMember, nextToken string, err error)
+	// ListAllAccessListMembers returns a paginated list of all access list members for all access lists.
```

Comment:
nit: IMO sounds simpler
```suggestion
	// ListAllAccessListMembers returns a paginated list of all members of all access lists.
```

What with duplicates? If a user is a member of multiple access lists will it be returned once or multiple times?

Reply:
It's not possible to have duplicates. The backend key contains the username